CIPs/0021.md

```diff
@@ -0,0 +1,46 @@
+# CIP [0021]: Governable lookback window
+
+- Date: 2020-09-25
+- Author: @andres-dg
```

Comment:
hi @andres-dg  thank you for submitting this CIP, I would like you to update it to follow the newly activated CIP process which has it's newly updated template here: https://github.com/celo-org/celo-proposals/blob/master/CIPs/cip-template.md

Also, if the CIP is in Draft status, I'd be able to merge the proposal into the repo after the other action items are completed.